README.md

```diff
@@ -356,6 +365,20 @@ Property | Type | Default | Description
 **data** | *Object* | `{}` | *inherited from [ActionConfig](#interface-actionconfig)*
 **abortOnFail** | *Boolean* | `true` | *inherited from [ActionConfig](#interface-actionconfig)*
 
+## Transform Function
```

Comment:
This feels a bit out of place. The rest of the docs at this level are named `actions` where as `transform` is a common prop of several actions. Maybe move this to a footnote under the "Built-In Actions" header and link to it from each action's property description? Something to keep in mind is that H1 and H2 headings in this file will show up in the main documentation navigation on the website.

Reply:
I wrote this part before working on the usage example files. I've moved it up and removed the code block, but if you want we can just get rid of it.

Reply:
I think it's beneficial. 👍